src/components/Board.vue

```diff
@@ -1,39 +1,54 @@
 <template>
-  <div id="game" ref="game">
-    <div id="user-name">
-      Username: {{ userEmail }}
+  <div class="board-wrapper">
+    <div id="game" ref="game">
+      <div id="user-name">
+        Username: {{ userEmail }}
+      </div>
+      <div id="score">
+        Score: {{ this.$store.state.scoreStore.score }}
+      </div>
+      <button
+        class="button-base"
+        @click="startGame"
+        :disabled="storeArrows.length !== 0"
+      >
+        start
+      </button>
+      <button
+        class="button-base mt-5"
+        @click="stopGame"
+        :disabled="storeArrows.length === 0"
+      >
+        stop
+      </button>
+      <areas @calculate-positions="setPositions" />
+      <arrow
+        v-for="n in storeArrows"
```

Comment:
I'd avoid not very meaningful names as `n`. It's ok as an index in loops, but it this case `:key="arrow.id"` has more sense as for me